tornjak-frontend/src/components/agent-create-join-token.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import axios from 'axios';
+import GetApiServerUri from './helpers';
+import IsManager from './is_manager';
+import {
+  serverSelectedFunc
+} from 'redux/actions';
+import { RootState } from 'redux/reducers';
+// import PropTypes from "prop-types";
+
+type CreateJoinTokenProp = {
+  globalServerSelected: string,
+}
+
+type CreateJoinTokenState = {
+  name: string,
+  ttl: number,
```

Comment:
nit: if there's an int type (number of bits) for the interface, can we set that number here as well

Reply:
The number data type stores as a double precision 64-bit number. And typescript has only two datatypes when it comes to numbers. The other one is bigint - used for very large numbers bigger than 9007199254740991. And we can always retain the precision to 64 bits with numbers.